01_easy_algorithms/04_sharing_is_caring/04_sharing_is_caring.rb

```diff
@@ -0,0 +1,4 @@
+def show_the_love(x)
```

Comment:
method parameter should be at least 3 chars long. Don't name it something ambiguous like 'x'. When you write code, write it keeping in mind that other developers might have to read through your code and figure out what it is.

Or that you might come back to your own code after a while and may have to figure it out yourself. Call it 'art' or 'input_arr' or something.